3_visualize/src/build_peaks_svg.R

```diff
@@ -35,4 +35,22 @@ build_peaks_svg <- function(out_file, data_in, sites_sf, svg_width, svg_height)
   
   xml2::write_xml(svg_root, file = out_file)
   
+  # Copy to vue
+  file.copy(out_file, 
+            sprintf("src/assets/anomaly_peaks_%s_%s.svg",
+                    gsub("-", "", min(data_in$Date)),
+                    gsub("-", "", max(data_in$Date))),  
+            overwrite = TRUE)
```

Comment:
It's good practice for each target function to only create one file. If you were just looking at the makefile, you would have no idea that a second file gets created here. Let's move this code chunk to its own function and be its own target.

Reply:
This is tricky though because of how we want to name these files dynamically. I wonder if moving and listing SVGs to a CSV is the best way.

Reply:
I've separated this function out into a new target now, and removed the dynamic naming for the time being. The reason for the dynamic naming was to be able to create an archive of past dates when we re-generate the viz, but this can be addressed later when we have sorted out how to store and access the data associated with past time periods, as I expect that will influence how this is done.